sdk/typescript/libraries/botbuilder-solutions/src/skills/skillsConfiguration.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright(c) Microsoft Corporation.All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { EnhancedBotFrameworkSkill } from './enhancedBotFrameworkSkill';
+
+/**
+ * A helper class that loads Skills information from configuration.
+ */
+export class SkillsConfiguration {
+    public readonly skillHostEndpoint: string;
+    public skills: Map<string, EnhancedBotFrameworkSkill> = new Map<string, EnhancedBotFrameworkSkill>();
+
+    public constructor(skills: EnhancedBotFrameworkSkill[], skillHostEndpoint: string) {
+        
+        if (skills !== null)
```

Comment:
It's better to have undefined instead of nulll
```suggestion
        if (skills !== undefined)
```